pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1208,11 +1208,11 @@ protected void handleProducer(final CommandProducer cmdProducer) {
                                         }
                                         } else {
                                             producer.closeNow(true);
-                                        log.info("[{}] Cleared producer created after connection was closed: {}",
-                                                remoteAddress, producer);
-                                        producerFuture.completeExceptionally(
-                                                new IllegalStateException(
-                                                        "Producer created after connection was closed"));
+                                            log.info("[{}] Cleared producer created after connection was closed: {}",
```

Comment:
Same here, when a connection is closed, we log all the producer that were cleared